src/color.cc

```diff
@@ -276,6 +505,49 @@ rgba_from_rgba(uint8_t r, uint8_t g, uint8_t b, uint8_t a) {
 }
 
 /*
+ * Helper function used in rgba_from_hsla().
+ */
+
+static float hue_to_rgb(float t1, float t2, float hue) {
+  if(hue < 0) hue += 6;
```

Comment:
style: the paren backing up against the `if` here is inconsistent, newlines after the `if` may also be required, look at the rest of  the project for prevailing code style (note: I can't tell you off the top of my head what the style should be, I'm just noticing inconsistencies in _this_ contribution and would like it to be made internally consistent and consistent with the rest of the existing code)